app/views/pages/api_samples/get_pubs_out.json

```diff
@@ -191,12 +191,12 @@
                 {
                     "id": "15137956",
                     "type": "PMID",
-                    "url": "http://www.ncbi.nlm.nih.gov/pubmed/15137956"
+                    "url": "https://www.ncbi.nlm.nih.gov/pubmed/15137956"
                 },
                 {
                     "id": "",
                     "type": "WoSItemID",
-                    "url": "http://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/"
+                    "url": "https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/"
```

Comment:
Yeah, there is no ID on that URL.  Not sure why its there (bad data?).  Not sure what these .json files are used for.  even the http one doesn't resolve.